src/Stream.php

```diff
@@ -60,7 +60,7 @@ public function __construct($stream, $mode = 'r')
     public function __toString()
     {
         if (! $this->isReadable()) {
-            return '';
+            throw new RuntimeException('Stream is not readable');
```

Comment:
I know; it was seeing it in the context of the method that made me remember why I wasn't throwing an exception previously here. Since the exception is non-recoverable, we must cast to an empty string instead.